app/models.py

```diff
@@ -978,6 +978,8 @@ class Notification(db.Model):
     created_by = db.relationship('User')
     created_by_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), nullable=True)
 
+    reply_to_text = db.Column(db.String, nullable=True)
```

Comment:
Update the serialise method to include this column.

Reply:
That data is not included now. It would change the fetch from GET /notifications